packages/network/wpa_supplicant/package.mk

```diff
@@ -28,6 +28,7 @@ PKG_SECTION="network"
 PKG_SHORTDESC="wpa_supplicant: An IEEE 802.11i supplicant implementation"
 PKG_LONGDESC="The wpa_supplicant is a free software implementation of an IEEE 802.11i supplicant. In addition to being a full-featured WPA2 supplicant, it also has support for WPA and older wireless LAN security protocols."
 PKG_TOOLCHAIN="make"
+PKG_BUILD_FLAGS="lto-parallel"
```

Comment:
`+lto-parallel`

Reply:
agreed to all suggestions, will update shortly